recipes/gemmlowp/all/test_package/test_package.cpp

```diff
@@ -1,11 +1,392 @@
-#include <cstdlib>
+// Example code illustrating the theory exposed in doc/quantization.md
```

Comment:
It could be simplified, don't need an entire example code, only few methods are enough to test a package.